### ISSUE TITLE
Please update ESV Protocol spec to document Certify json fields

### ISSUE THREAD

**Issue by rruss-ldos** [usnistgov/ESV-Server#42]
The json fields used for Certify requests are not currently documented like the Registration fields are. Although largely self-explanatory, please update [section 5 of the protocol spec](https://github.com/usnistgov/ESV-Server/blob/main/Entropy%20Source%20Validation%20Protocol.md#5-certify) to explain these fields.

Thank you

**Comment by celic** [comment]
@rruss-ldos this should be fixed. Take a look and see if you have any further questions.

**Comment by rruss-ldos** [comment]
@celic Yes, this is largely what I had in mind. For limitEntropyAssessmentToSingleModule it might be better to also give its web-portal equivalent.



limitEntropyAssessmentToSingleModule | boolean stating whether the entropy assessment associated with this certification is applicable to a single module / vendor. Analogous to "Reuse restricted to vendor" | boolean
-- | -- | --




or


limitEntropyAssessmentToSingleModule | boolean stating whether the use of the entropy assessment associated with this certification is restricted to a single module / vendor | boolean
-- | -- | --




Thanks
